### PR TITLE
Added Compose demo screen for spacing

### PR DESCRIPTION
**Pull request by jaanainkinen**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import net.skyscanner.backpack.compose.tokens.BpkColor
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.data.Token
+import net.skyscanner.backpack.demo.data.values
+
+@Composable
+@Preview
+fun SpacingComposeStory() {
+  LazyColumn() {
+    val spacingData: List<Token<Dp>> = BpkSpacing.values.sortedBy { it.value }
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt:46]: Let's rename it to data to be consistent

File: app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+private fun SpacingSampleRow(token: Token<Dp>) {
+  Row(
+    modifier = Modifier
+      .height(56.dp)
+      .padding(BpkSpacing.Base),
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    BpkText(
+      text = "${token.name} = ${token.value}",
+      modifier = Modifier
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt:63]: Could you please make it single line and add a trailing comma?

**Review by bvitaliyg — APPROVED** [review]
💯

File: app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt (hunk 2 above)
Comment (jaanainkinen) [reply app/src/main/java/net/skyscanner/backpack/demo/compose/SpacingStory.kt]: Done!

**Pull request merged**